### PR TITLE
Chapter 6 Lab: Turtle Simulator

### PR DESCRIPTION
**Pull request by AnuAlli**

### CODE REVIEW

**Comment by khatchad** [comment]
I don't have a submission in Blackboard for your group. Please note the submission instructions:

> ## Submission
> 1. Modify the files and [**commit**](http://gitref.org/basic/#commit) changes to complete your solution.
> 2. [**Push**](http://gitref.org/remotes/#push)/sync the changes up to GitHub.
> 3. [Create a **pull request**](https://help.github.com/articles/creating-a-pull-request) on the original repository to turn in the assignment.
> 4. Submit, in Blackboard, the GitHub username corresponding to the pull request.

**Comment by khatchad** [comment]
I already have a graded attempt for you.

**Comment by khatchad** [comment]
OK I see that the first attempt is before you were in a group. Reopened.

**Comment by khatchad** [comment]
This submission doesn't compile. As such, I can't grade it. Automatic grade of 0.

**Comment by AnuAlli** [comment]
I created a pull request ,and submitted my github username through blackboard has you requested.

**Comment by khatchad** [comment]
Actually, this does compile. Sorry about that.

**Comment by AnuAlli** [comment]
it’s Okay, and thanks for the understanding.

File: src/cst1201/Turtle.java [hunk 1]
```diff
@@ -117,6 +117,10 @@ private Color getColorFromString(String c) {
         return Color.BLACK; //return black otherwise
     }
 
+    void backward(int i) {
```
Comment (khatchad) [inline src/cst1201/Turtle.java:120]: You were not to change the `Turtle` class.

File: src/cst1201/IntialDrawing.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+public class IntialDrawing {
+    
+    
+     public static void main (String[] args) {
+         
+         IntialDrawing a = new IntialDrawing();
+         a.draw();
+     }
+   
+     public void draw(){
```
Comment (khatchad) [inline src/cst1201/IntialDrawing.java:23]: > [2pt] Your InitialDrawing class can draw your initials wth a plus sign

:-1: -1 pt. No plus sign.

File: src/cst1201/Bowtie.java [hunk 3]
```diff
@@ -3,8 +3,42 @@
...
+   
+    
+//this is the bowtie class header of the draw() method it will implement the algorithm to draw a bowtie below.
+  
+  public void draw(){
+      
+      
+      
+   
+  Turtle myTurtle = new Turtle();
```
Comment (khatchad) [inline src/cst1201/Bowtie.java:18]: :-1: 

> [1pt] Good programming style -- including proper indentation is expected

**Comment by AnuAlli** [comment]
I changed the classes in order to differentiate between them.

**Pull request closed without merging**